src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs

```diff
@@ -115,7 +115,7 @@ internal static OtlpTrace.Span ToOtlpSpan(this Activity activity)
             foreach (var kvp in activity.Tags)
             {
                 var attribute = ToOtlpAttribute(kvp);
-                if (attribute != null)
+                if (attribute != null && attribute.Key != SpanAttributeConstants.StatusCodeKey && attribute.Key != SpanAttributeConstants.StatusDescriptionKey)

```

Comment:
please unit test this part.

Reply:
This part is already unit tested as a part of `ToOtlpResourceSpansTest`.
```
            Assert.Equal(OtlpTrace.Status.Types.StatusCode.NotFound, otlpSpan.Status.Code);
            Assert.Equal(Status.NotFound.Description ?? string.Empty, otlpSpan.Status.Message);
            Assert.Empty(otlpSpan.Attributes);
```
`Tags` are converted to attributes in otlp, in this test case even after adding the special tags attributes is empty. 
Should we create a new test case only to validate this scenario?